CHANGELOG.md

```diff
@@ -6,12 +6,15 @@
 # Changelog
 
 <!-- next version -->
+
 ## Unreleased
 
 ### Breaking
 
 ### Features
 
+- Add semantic conventions for recording exceptions in events [#970](https://github.com/open-telemetry/semantic-conventions/pull/970)
```

Comment:
We are not updating changelog directly. Please check requirements in the description of your PR